.github/workflows/beam_PreCommit_Java.yml

```diff
@@ -26,7 +26,6 @@ on:
       - 'examples/kotlin/**'
       - 'release/**'
       - '.github/workflows/beam_PreCommit_Java.yml'
-      - '!sdks/java/extensions/avro/**'
```

Comment:
java precommit runs `sdks:java:extensions:avro:build` so we should not exclude it